CMakeLists.txt

```diff
@@ -668,7 +668,7 @@ target_include_directories(re-objs PRIVATE
   ${OPENSSL_INCLUDE_DIR} ${ZLIB_INCLUDE_DIRS})
 target_include_directories(re-objs PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-    $<INSTALL_INTERFACE:include>
+    $<INSTALL_INTERFACE:include> # pointless, object libs don't get installed.
```

Comment:
Should we remove this line?

Reply:
IMHO yes. Unless there is any effect which I'm not aware of. But IIRC your project only uses the target objects.